README.md

```diff
@@ -7,25 +7,23 @@ This repo contains the code for the [Fishbrain Engineering Blog](https://enginee
 Run:
 
 ```sh
-> yarn && yarn dev
+> npm i
+> npm run dev
```

Comment:
Isn't it possible to use yarn?

Reply:
Yeah, it could be. Is NPM an issue though? At least for the Node services owned by the Platform Team we've been trying to consolidate on using NPM. But given that technically this service is owned by the Web Guild (though it hasn't been getting maintained) I'd be happy to switch it back to Yarn.

Reply:
fyi @agierloff88 as we were talking about updating Yarn. It was a fairly basic process, though I'm not certain yet how it will behave when switching between projects using differing versions of Yarn. It added config files, but I'm not certain if v1 will support it or not.

Reply:
I looked into it more and the actually recommend committing yarn itself (it's in the `.yarn` directory). That feels a little whack to me, but let's see how it goes I suppose 🤷